docs/api/client.rst

```diff
@@ -0,0 +1,125 @@
+###############
+Client endpoint
+###############
+
+.. _client-endpoint:
+
+Websocket endpoint
+==================
+
+**Requires authentication**
+
+Clients connects the websocket directly on ``/``
+
+::
+
+    ws://localhost:8765/
+
+
+Creating a new worker
+=====================
+
+In order to ask for a worker creation client need to send this first standza:
```

Comment:
I'm not sure we should use the term "standza" (which is stanza, anyway). https://en.wikipedia.org/wiki/Stanza_%28disambiguation%29